lib/manageiq/providers/workflows/builtin_methods.rb

```diff
@@ -2,9 +2,10 @@ module ManageIQ
   module Providers
     module Workflows
       class BuiltinMethods < BasicObject
-        def self.email(params = {}, _secrets = {}, _context = {})
+        def self.email(params = {}, _secrets = {}, context = {})
           options = params.slice("To", "From", "Subject", "Cc", "Bcc", "Body", "Attachment").transform_keys { |k| k.downcase.to_sym }
           options[:from] ||= ::Settings.smtp.from
+          options[:to]   ||= context.execution["_requester_email"] if context.respond_to?(:execution)
```

Comment:
When would context not respond to execution?  Do we actually ever pass in an actual hash or always a `Floe::Context` object?

Reply:
I think we sometimes pass the context in as a `Hash`.

Reply:
Yeah if it can be a hash I think we should `context.dig("Execution", "_requester_email")` which should work with either, but it'd be better if we stay consistent with the types that we're passing in.

Reply:
we only ever pass in a Context, but the tests pass nothing, so they have a hash there.

Updated tests to pass a context (and match the way the methods are actually used)